src/main/java/com/nionios/trial/api/TalentEndpoint.java

```diff
@@ -26,22 +19,51 @@
     @Autowired
     private TalentService talentService;
 
+   @Autowired
+   private TalentManagerService talentManagerService;
+
+
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response addTalent(Talent talent){
+        Talent result = talentManagerService.addTalent(talent);
+        return Response.accepted(result).build();
+    }
+
+    @DELETE
+    @Path(value = "/{id}")
```

Comment:
`value = ` is not necessary in @Path apparently, can you remove that from each @Path?

Reply:
Done